policy/release/attestation_type.rego

```diff
@@ -19,9 +23,10 @@ import data.lib
 #     known_attestation_types:
 #     - https://in-toto.io/Statement/v0.1
 #
-deny[result] {
-	att := lib.pipelinerun_attestations[_]
+deny contains result if {
+	some att in lib.pipelinerun_attestations
```

Comment:
I think `some` is right because if we have multiple attestations and _some_ are invalid, we want the deny rule to trigger. If we used `every`, then it would only trigger the deny rule if _all_ the attestations are invalid, not just _some_.

It's a bit strange because we have to think in terms of what qualifies as denied instead of allowed.

Reply:
OK, I read it wrong, I really like that syntax.